tests/publiccloud/azure_cli.pm

```diff
@@ -36,17 +36,21 @@ sub run {
     my $resource_group = "openqa-cli-test-rg-$job_id";
     my $machine_name = "openqa-cli-test-vm-$job_id";
 
+    my $openqa_ttl = get_var('MAX_JOB_TIME', 7200) + get_var('PUBLIC_CLOUD_TTL_OFFSET', 300);
+    my $created_by = get_var('PUBLIC_CLOUD_RESOURCE_NAME', 'openqa-vm');
+    my $tags = "openqa-cli-test-tag=$job_id openqa_created_by=$created_by openqa_ttl=$openqa_ttl";
+
+    # Configure default location and create Resource group
     assert_script_run("az configure --defaults location=southeastasia");
-    assert_script_run("az group create -n $resource_group");
+    assert_script_run("az group create -n $resource_group --tags '$tags'");
 
+    # Pint
```

Comment:
well I would do one of two things : 
1. drop this comment 
2. add something more detailed like `Pint - command line tool to query pint.suse.com ( info about ALL images in ALL CSP's managed by SUSE) `
In current form the one who know what is pint don't need this line , the one who don't know what is pint can not get any useful info from this line

Reply:
I changed it to:
```
# Pint - command line tool to query pint.suse.com to get the current image name
```